tests/50federation/30room-join.pl

```diff
@@ -953,7 +953,110 @@ sub assert_is_valid_pdu {
 
             # XXX currently synapse fails with a 500 here. I'm not really convinced that's
             # a thing we want to enforce, but we don't really have a specced way to say
-            # "a remote server did something wierd".
+            # "a remote server did something weird".
+            Future->done(1);
+         }),
+      )
+   };
+
+test "Outbound federation event with an invalid signature should not break",
+   requires => [ local_user_fixture(), $main::INBOUND_SERVER,
+                 federation_user_id_fixture() ],
+
+   do => sub {
+      my ( $user, $inbound_server, $creator_id ) = @_;
+
+      my $local_server_name = $inbound_server->server_name;
+      my $datastore         = $inbound_server->datastore;
+
+      my $room_alias = "#50fed-room-alias:$local_server_name";
```

Comment:
this probably needs to be unique; I think this alias is used elsewhere in the test suite?

Reply:
It probably should be. Is there any pattern or something I should use here? Or should I be using the fixture that generates a room alias?

Reply:
well there was some idea of making the alias reflect the name of the test file, but clearly it's not very closely followed.

The trouble with the fixture (iirc) is that it wants to generate an alias that is suitable for the system under test rather than for the sytest federation server.

Reply:
I made it unique then.